packages/graph/hash_graph/Makefile.toml

```diff
@@ -19,3 +19,7 @@ private = false
 extend = "task"
 args = ["nextest", "run", "--cargo-profile", "${CARGO_MAKE_CARGO_PROFILE}", "@@split(CARGO_TEST_FLAGS, )", "--workspace", "--test", "integration", "--profile", "integration", "${@}"]
 dependencies = ["install-nextest"]
+
+[tasks.test-rest-api]
```

Comment:
Just for anyone else reviewing/looking at this. This isn't added to any CI, just a local dev command for `cargo make` for now :+1: